build.xml

```diff
@@ -23,6 +23,25 @@
         </javac>
     </target>
 
+    <target name="test" depends="compile" description="run all tests">
+        <mkdir dir="${test.result.dir}"/>
+        <junit printsummary="on"
+               haltonfailure="no"
+               errorProperty="tests.failed"
+               failureProperty="tests.failed">
```

Comment:
Please add  dir="${src.dir}/test/resources" to junit tag which is specified in the existing <junit />.

To avoid this kind of mistakes, we'd better to have "sub routine" way in the future.

Reply:
Thanks. I've updated.